I/IntelOpenMP/build_tarballs.jl

```diff
@@ -3,63 +3,68 @@ using BinaryBuilder
-    ),
-    "x86_64-w64-mingw32" => (
-        "https://anaconda.org/intel/openmp/2018.0.3/download/win-64/openmp-2018.0.3-intel_0.tar.bz2" =>
-        "0aee3d9debb8b1c2bb9a202b780c2b2d2179e4cee9158f7d0ad46125cf6f3fa2"
-    ),
-    "i686-w64-mingw32" => (
-        "https://anaconda.org/intel/openmp/2018.0.3/download/win-32/openmp-2018.0.3-intel_0.tar.bz2" =>
-        "86ed603332ed7b4004e8a474943468589b222ef16d0d9aaf3ebb4ceaf743a39d"
-    ),
-    "x86_64-apple-darwin14" => (
-        "https://anaconda.org/intel/openmp/2018.0.3/download/osx-64/openmp-2018.0.3-intel_0.tar.bz2" =>
-        "110b94d5ff3c4df66fc89030c30ad42378da02817b3962f14cb5c268f9d94dae"
-    ),
-)
-sources = [
-    source_dict[target],
+sources_macos = [
+    "https://anaconda.org/intel/openmp/2018.0.3/download/osx-64/openmp-2018.0.3-intel_0.tar.bz2" =>
+    "110b94d5ff3c4df66fc89030c30ad42378da02817b3962f14cb5c268f9d94dae"
 ]
 
+sources_linux32 = [
+    "https://anaconda.org/intel/openmp/2018.0.3/download/linux-32/openmp-2018.0.3-intel_0.tar.bz2" =>
+    "f06edc0c52337658fd4b780d0b5c704b0ffb1c156dced7f5038c1ebbda3d891b",
+]
+
+sources_linux64 = [
+    "https://anaconda.org/intel/openmp/2018.0.3/download/linux-64/openmp-2018.0.3-intel_0.tar.bz2" =>
+    "cae3ef59d900f12c723a3467e7122b559f0388c08c40c332da832131c024409b",
+]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir
 if [[ ${target} == *mingw* ]]; then
-    mv Library/bin ${prefix}/bin
+    mv Library/bin ${libdir}
 else
-    mv lib ${prefix}/lib
+    mv lib ${libdir}
 fi
-mkdir -p ${prefix}/share
-mv info/*.txt ${prefix}/share/
+install_license $WORKSPACE/srcdir/info/LICENSE.txt
 """
 
-# These are the platforms we will build for by default, unless further
-# platforms are passed in on the command line
-platforms = [platform_key_abi(target)]
-
 # The products that we will ensure are always built
-products(prefix) = [
-    LibraryProduct(prefix, "libiomp5", :libiomp),
+products = [
+    LibraryProduct(["libiomp5", "libiomp5md"], :libiomp),
 ]
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
 ]
 
-# Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; skip_audit=true)
+# Install first for win32, then win64.  This will accumulate files into `products` and also wrappers into the JLL package.
+non_reg_ARGS = filter(arg -> arg != "--register", ARGS)
+
+include("../../fancy_toys.jl")
+
+if should_build_platform("i686-w64-mingw32")
+    build_tarballs(non_reg_ARGS, name, version, sources_win32, script, [Windows(:i686)], products, [])
+end
+if should_build_platform("x86_64-w64-mingw32")
+    build_tarballs(non_reg_ARGS, name, version, sources_win64, script, [Windows(:x86_64)], products, [])
+end
+if should_build_platform("x86_64-apple-darwin14")
+    build_tarballs(non_reg_ARGS, name, version, sources_macos, script, [MacOS(:x86_64)], products, [])
```

Comment:
I'm not skipping audit, only for macOS there is an error (`ERROR: Failed to run /opt/x86_64-apple-darwin14/bin/install_name_tool!`)

Reply:
Uhhh, that's not good! It should be able to run `install_name_tool`

Reply:
Oh I see; it's because they're not real files.  That's fine.